agent/src/jvmMain/kotlin/com/epam/drill/agent/websocket/WsClient.kt

```diff
@@ -27,12 +28,20 @@ import com.epam.drill.common.message.DrillMessageWrapper
 import com.epam.drill.common.message.Message
 import com.epam.drill.common.message.MessageType
 
-object WsClient : WsClientReconnectHandler {
+object WsClient : WsClientReconnectHandler, RetentionCallbacks {

```

Comment:
These callbacks have nothing to do with coverage retention task.
Rename to something like `ConnectionsStatusCallbacks`

Reply:
done